x/tx/signing/textual/handler.go

```diff
@@ -163,7 +163,7 @@ func (r *SignModeHandler) GetMessageValueRenderer(md protoreflect.MessageDescrip
 func (r *SignModeHandler) init() {
 	if r.scalars == nil {
 		r.scalars = map[string]ValueRendererCreator{}
-		r.scalars["cosmos.Int"] = func(fd protoreflect.FieldDescriptor) ValueRenderer { return NewIntValueRenderer(fd) }
+		r.scalars["cosmos.Int"] = NewIntValueRenderer
```

Comment:
nice catch, did the linter find this?

Reply:
Yes, one of the linters caught this -- I don't recall which one offhand.